tests/server/blueprints/variants/test_variants_views.py

```diff
@@ -254,6 +254,27 @@ def test_str_variants(app, institute_obj, case_obj):
         assert resp.status_code == 200
 
 
+def test_fusion_variants(app, institute_obj, case_obj):
```

Comment:
Nice with this tests. I see that you are using case_obj (the demo case witch is a WES case) in the testing. I think it would be more appropriate to use the other demo case, the rna fusion case. What happens in this test is that you are passing WES variants to the fusion variants page, not fusion variants. I'll see if it's already available in the test fixtures so we can use it!